src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -311,9 +322,10 @@ public static FeedSource checkFeedSourcePermissions(Request req, FeedSource feed
         }
         String orgId = feedSource.organizationId();
         boolean authorized;
+
         switch (action) {
             case CREATE:
-                authorized = userProfile.canAdministerProject(feedSource.projectId, orgId);
+                authorized = userProfile.canAdministerProject(feedSource.id, orgId);
```

Comment:
This seems like it should not have been changed.

Reply:
Do not know how that happened! Fixed in 9ac824a